src/desktop/main.cc

```diff
@@ -500,61 +535,61 @@ MAIN {
       return;
     }
 
-    if (cmd.name == "hide") {
+    if (message.name == "hide") {
       window->hide(EMPTY_SEQ);
       return;
     }
 
-    if (cmd.name == "inspect") {
+    if (message.name == "inspect") {
       window->showInspector();
       return;
     }
 
-    if (cmd.name == "background") {
+    if (message.name == "background") {
```

Comment:
we could make this message name less ambiguous while we're here

Reply:
There was two different `cmd` variables:
- `<os>_cmd` string
- command from webvew process or backend process
I think that message is a good name for this IPC thing, but I'm open to other alternatives

Reply:
I like `ipcMessage` as well